gtfs/spec/en/reference.md

```diff
@@ -29,174 +29,176 @@ This document explains the types of files that comprise a GTFS transit feed and
 
 This section defines terms that are used throughout this document.
 
-* **Field required** - The field column must be included in your feed, and a value must be provided for each record. Some required fields permit an empty string as a value. To enter an empty string, just omit any text between the commas for that field. Note that 0 is interpreted as "a string of value 0", and is not an empty string. Please see the field definition for details.
-* **Field optional** - The field column may be omitted from your feed. If you choose to include an optional column, each record in your feed must have a value for that column. You may include an empty string as a value for records that do not have values for the column. Some optional fields permit an empty string as a value. To enter an empty string, just omit any text between the commas for that field. Note that 0 is interpreted as "a string of value 0", and is not an empty string.
-* **Field conditionally required** - This field or file is **required** under certain conditions, which are outlined in the field or file *Details*. Outside of these conditions, this field or file is optional. 
-* **Dataset unique** - The field contains a value that maps to a single distinct entity within the column. For example, if a route is assigned the ID **1A**, then no other route may use that route ID. However, you may assign the ID **1A** to a location because locations are a different type of entity than routes.
+* **Feed** - Successive datasets comprise a feed.
+* **Dataset** - A complete set of files defined by this specification reference. Datasets should be published at a public, permanent URL, including the zip file name. (e.g., www.agency.org/gtfs/gtfs.zip). For more information see GTFS best practices. 
+* **Field** - A specific type of information for records in a .txt file. Represented, in a table, as a column.
```

Comment:
> A specific type of information

I don't get it.

I think I would define record, then field, then field value.

Reply:
IMHO a "field" is a property of some object or entity, it's not a type of information.

Reply:
Agreed that Field and Field Value definitions should be together.  Personally, I'd put Record after these two, but that's debatable.

Reply:
> > A specific type of information
> 
> I don't get it.

You're right, the wording needs to changed. "Specific type of information" is supposed to get at the fact that all the field values of a particular field describe a specific property of some object . In other words, the field `stop_lat` always contains a specific type of geospatial information about a location. Perhaps this doesn't need to be included, but then I'm not sure what we should say about fields other than the fact that they are often represented as columns. 

Any ideas on how we should define "field"?